### PR TITLE
Add missing installation options to packaging/installer/README.md

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

This PR adds the missing installation options from `packaging/installer/methods` to `packaging/installer/README.md`.

### CODE REVIEW

File: packaging/installer/README.md [hunk 1]
```diff
@@ -107,28 +107,56 @@ troubleshooting with Netdata.
...
+    os="Install with native packages"  
+    svg="package" />
+  <InstallBoxRegexLink
+    to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/pfsense.md)"
+    os="Install on pfSense"
+    svg="pfsense" />
+  <InstallBoxRegexLink
+    to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/synology.md)"
+    os="Install on Synology"
+    svg="synology" />
```
Comment (ilyam8) [inline packaging/installer/README.md:159]: Do we want to keep methods that we don't actually support/don't even know if they work (I bet they don't)? cc @netdata/agent-sre 

I guess there were missing for a reason.

Comment (Ancairon) [reply packaging/installer/README.md:159]: Well, that's actually a good question, if we don't support them, then we need to mention it in their markdown files and obviously not include them, or something similar
@cakrit

Comment (cakrit) [reply packaging/installer/README.md:159]: Everything that exists under packaging/installer/methods is supported, either by Netdata or someone else.
If something is no longer supported, it needs to be removed from the repo. @Ferroin @tkatsoulas plz verify the list and make any necessary changes to the files in the repo and to this document.

**Review by cakrit — APPROVED** [review]

**Comment by cakrit** [comment]
We decided to leave this page just mention the most popular options. We'll just add a link at the end to the rest of them.

**Pull request closed without merging**